martin/src/pmtiles/mod.rs

```diff
@@ -361,3 +390,13 @@ impl PmtFileSource {
         Self::new_int(id, path, reader).await
     }
 }
+
+/// Interpret an environment variable as a [`bool`]
+///
+/// This ignores casing, but interprets bad utf8 encoding as `false`.
+fn get_env_as_bool(key: &'static str) -> bool {
+    let env_val = std::env::var_os(key)
+        .unwrap_or_default()
+        .to_ascii_lowercase();
+    [Some("1"), Some("true")].contains(&env_val.to_str())
```

Comment:
a bit longer, but feels more "rusty"

```suggestion
    std::env::var_os(key)
        .unwrap_or_default()
        .to_ascii_lowercase()
        .to_str()
        .is_some_and(|val| matches!(val, "1" | "true"))
```